_data/datasets/budget-analysis-for-advancing-socio-economic-rights.yaml

```diff
@@ -1,62 +0,0 @@
-author:
```

Comment:
This is getting deleted because the dataset was renamed to use socioeconomic as one word.